stonesoup/runmanager/tests/test_csvs/detections.csv

```diff
@@ -1,3 +1,8 @@
 time,x,y
+<<<<<<< HEAD
```

Comment:
This merge conflict needs to be resolved. The lines between <<<< and ==== are your local branch and the lines between ==== and >>>> are introduced from another commit.